### DISCUSSION TITLE
[A-L] (30 maggio 2024) Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/progettazione-di-algoritmi#105] [Esercitazioni A-L]
![immagine](https://github.com/user-attachments/assets/2c48098c-4746-497c-b0f7-daaaf9177caf)

**Comment by Elia-Belli** [comment]
**Legenda**
- In **Rosso** l'alborescenza di visita
- In **Blu** gli archi all'indietro
- In **Verde** gli archi di attraversamento
<p align= center>
<img width = 500 src= https://github.com/user-attachments/assets/73c4932a-77f3-41a6-86f7-8616bf825dc8>
</p>

**Comment by CiottoloMaggico** [comment]
![immagine](https://github.com/user-attachments/assets/90286427-51c0-495f-a4a5-81e9b09fd08e)

**Comment by daromacarlo** [comment]
![immagine](https://github.com/user-attachments/assets/4da7a9b9-9da1-4240-a489-0af999211c4a)

**Comment by cek753** [comment]
<img width="2045" height="1314" alt="image" src="https://github.com/user-attachments/assets/00ef43d2-87d9-4fa4-987c-952bb9ad3e76" />
